lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java

```diff
@@ -27,12 +26,7 @@
 import org.esbtools.eventhandler.lightblue.client.LightblueErrors;
 import org.esbtools.eventhandler.lightblue.client.LightblueRequester;
 import org.esbtools.eventhandler.lightblue.client.UpdateRequests;
-import org.esbtools.eventhandler.lightblue.locking.LockNotAvailableException;
 import org.esbtools.eventhandler.lightblue.locking.LockStrategy;
```

Comment:
Removed LockStrategy as constructor parameter. Corresponding code changes in internal event-handler implementation.